### PR TITLE
Renaming 'fields' to 'selects'

### PR DESCRIPTION
**Pull request by mssfang**
Rename all 'fields' to selects
fix ComplexConfiguration's wrong package path.

### CODE REVIEW

File: sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/SettingSelector.java [hunk 1]
```diff
@@ -202,18 +202,18 @@ public SettingSelector setRange(Range range) {
 
     @Override
     public String toString() {
-        String fields;
-        if (ImplUtils.isNullOrEmpty(this.fields)) {
-            fields = "ALL_FIELDS";
+        String selects;
+        if (ImplUtils.isNullOrEmpty(this.selects)) {
+            selects = "SELECT_ALL";
```
Comment (alzimmermsft) [inline sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/SettingSelector.java]: @conniey Do you remember if `ALL_FIELDS` triggered the service to return something special? Looking at their documentation there is no mention of this and we likely should pass nothing in this case to get the default selection.

File: sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/SettingSelects.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
  * @see SettingSelector
  * @see ConfigurationAsyncClient
  */
-public enum SettingFields {
+public enum SettingSelects {
```
Comment (alzimmermsft) [inline sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/SettingSelects.java]: Maybe name this to `FieldSelections`? We should bring this naming up in the stand-up

Comment (alzimmermsft) [inline sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/SettingSelects.java]: Let's change this from being an `enum` to `ExpandableStringEnum` to help make this version independent.

Comment (conniey) [reply sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/SettingSelects.java]: There's a teams discussion in the azconfig channel about this. I think the name should be agreed on by all language folks before updating.

**Comment by mssfang** [comment]
As AzConfig Channel discussed. All languages will stick with "fields". So no change needed.

**Pull request closed without merging**